haas/api.py

```diff
@@ -605,20 +702,29 @@ def show_network(network):
 
     network = _must_find(model.Network, network)
 
-    if network.access is not None:
-        auth_backend.require_project_access(network.access)
+    if network.access:
+        #auth tests: admin owned networks with a project access
+        #TODO only network owner allowed to show the network?
```

Comment:
we discussed briefly and ended up with the code as it is currently so it will stay that way unless someone objects
I left the TODO so reviewers know that they might want to object